src/utils_stackdriver_json_test.c

```diff
@@ -83,9 +86,12 @@ DEF_TEST(time_series_summary) {
 
 DEF_TEST(collectd_time_series_response) {
   char buf[10000];
+  ssize_t file_size;
   int ret;
-  OK(read_file_contents(SRCDIR "/src/collectd_time_series_response_test.json", buf,
-                        sizeof(buf)) >= 0);
+  file_size = read_file_contents(SRCDIR "/src/collectd_time_series_response_test.json", buf,
+                                 sizeof(buf));
```

Comment:
This needs to be `sizeof(buf) - 1`, else you're at a risk of buffer overrun in line 94.

Reply:
Addressed by #174.